### ISSUE TITLE
Turn off zoom-in animation when switching photos

### ISSUE THREAD

**Issue by tomscii** [LycheeOrg/Lychee-v3#154]
### Detailed description of the problem

Lychee is great, but after a very short time using it, I found an annoying issue.

Quickly going through the photos with the left/right arrow (looking at each photo in turn), the zoom-in animation on showing each new photo is incredibly distracting and annoying. I would like to have absolutely no animation between two photos. I did not find any setting to disable this animation.

### Steps to reproduce the issue

Just keep pressing the left or right arrow in photo view mode. Your head will start geting dizzy from the zoom-in effect after a while.

### Output of the diagnostics (Settings => Diagnostics)

Diagnostics
-----------
Warning: Dropbox import not working. No property for dropboxKey.
No critical problems found. Lychee should work without problems!

System Information
------------------
Lychee Version (json):  3.2.7
DB Version:             update_030207
System:                 Linux
PHP Version:            7.2
MySQL Version:          50505
Imagick:                1
Imagick Active:         1
Imagick Version:        1799
GD Version:             bundled (2.1.0 compatible)
Plugins:

### Browser and system

Safari 12.02 on Mac OS 10.12.6 on a MacBook Pro 2017.
Lychee itself is installed on an armhf board running armbian linux.

**Comment by ildyria** [comment]
This is done through CSS.

Also related: https://github.com/LycheeOrg/Lychee/issues/51

**Comment by tomscii** [comment]
I brute-forced this by just killing what I thought was relevant from the CSS:
https://github.com/tomszilagyi/Lychee-front/commit/6746c52dc50d9b988e2c4cee59b07a79f488a85f

I am no frontend dev, so the above might be complete garbage, but after rebuilding and redeploying the site according to the docs, it solves my problem.

Having an on/off switch under Settings (similar to the one labeled "Use justified layout") or a drop-down selector in case of multiple options would be a perfect feature, if such a thing is feasible.

**Comment by ildyria** [comment]
It is indeed the part to delete. I don't think we will add such option as css modification is quite expensive.
I would rather consider adding a css field in the config where the user can add their own css e.g. in your case: 
```css
#image {
  transition: none !important;
}
```

**Comment by tomscii** [comment]
That would also be great -- of course it does not have to be exposed in the Settings menu, as long as there is an easy (=documented) way to change this in the config.

**Comment by tomscii** [comment]
Edit:
Problem solved, sorry for the noise!
I discovered the FAQ entry regarding the same issue and the CSS snippet provided there works fine!
Lychee rocks!

* * *

Hi,

Thank you for your feedback.

I just tested your CSS snippet (quoted below) on a fresh install of Lychee 3.2.16, by pasting it into the "Personalize your CSS" box in Settings.

It still does not remove the annoying zoom-in effect. I double-checked that I flushed all my browser caches and also tested with a fresh browser that has never seen this Lychee instance.

Would be great if I could upgrade to the latest Lychee without having to patch Lychee-frontend the way I did it up to now (tomszilagyi/Lychee-front@6746c52)

Any ideas?

> ```css
> #image {
>   transition: none !important;
> }
> ```
